51-dracut-rescue-postinst.sh

```diff
@@ -58,11 +65,18 @@ fi
 
 if [[ ! -f $NEW_KERNEL_IMAGE ]]; then
     cp --reflink=auto "$KERNEL_IMAGE" "$NEW_KERNEL_IMAGE"
+    [[ ! -f $NEW_KERNEL_HMAC  ]] && cp --reflink=auto "${KERNEL_IMAGE%/*}/.vmlinuz-${KERNEL_VERSION}.hmac" "$NEW_KERNEL_HMAC"
```

Comment:
hmm, check for existence of the source or discard the cp error messages

Reply:
Sure thing, I added a check for existence of the source.